book/homework_1_solutions.ipynb

```diff
@@ -0,0 +1,3405 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.legend.Legend at 0x7f75ec615730>"
+      ]
+     },
+     "execution_count": 45,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create a plot that uses the colors assigned to the table of the Atomic Mass vs the Melting Point.\n",
+    "plt.figure()\n",
+    "plt.xlabel('Atomic Mass')\n",
```

Comment:
Might be interesting to have them do subplots and to also add ionization energy.

Reply:
Are you thinking something like this?
![comparison_fig](https://user-images.githubusercontent.com/43828519/116100289-c538ff80-a67a-11eb-9bb2-3ac86c1048b4.JPG)

Would you like it added as a 4th exercise or as a modification of the third?

Reply:
extension of third please. Make sure to update the x axis label!

Reply:
To clarify, do you want it as atomic mass vs ionization energy or melting point vs ionization energy?

Reply:
Exercise 3 has been changed to include a subplot of the Ionization Energy vs the Melting Point.